external/cordova

```diff
@@ -1 +1 @@
-Subproject commit ef4383561e127521449aeb8da90806f6ea6f699d
+Subproject commit a67bddbd821cebe7adb7e284592a09df45c9cc37
```

Comment:
What the diff doesn't show is all that new magic in this submodule with new ```Cordova``` ;-)